frontend/src/components/GeyserStakeView.tsx

```diff
@@ -91,7 +176,33 @@ export const GeyserStakeView: React.FC<Props> = () => {
         onClick={handleGeyserInteraction}
         displayText={isStakingAction ? `Stake` : `Unstake`}
       />
-      {!isStakingAction && <UnstakeConfirmModal parsedUserInput={parsedUserInput} open={unstakeConfirmModalOpen} onClose={() => setUnstakeConfirmModalOpen(false)} onConfirm={handleConfirmUnstake}/>}
+      {!isStakingAction && (
+        <UnstakeConfirmModal
+          parsedUserInput={parsedUserInput}
+          open={unstakeConfirmModalOpen}
+          onClose={() => setUnstakeConfirmModalOpen(false)}
+          onConfirm={handleConfirmUnstake}
+        />
+      )}
+      {isStakingAction ? (
```

Comment:
nit: i think this flow is a little confusing to follow -- are we using isStakingAction as a triplet? (null, true, false)?

Reply:
it's used only as true, false